### PR TITLE
Update .gitignore for appsre pipeline

### PR DESCRIPTION
**Pull request by 2uasimojo**

### CODE REVIEW

**Comment by jharrington22** [comment]
This is to fix a failing build.

The local git repo isn't clean because boilerplate make docker-login creates a .docker dir right before `isclean` https://github.com/openshift/boilerplate/blob/master/boilerplate/openshift/golang-osd-operator/standard.mk#L104

**Review by jharrington22 — APPROVED** [review]

File: .gitignore [hunk 1]
```diff
@@ -86,3 +87,9 @@ tags
 
 # used with direnv
 .envrc
+
+### boilerplate
+# appsre pipeline
+Dockerfile.olm-registry
+saas-configure-alertmanager-operator-bundle/
```
Comment (2uasimojo) [inline .gitignore:94]:
```suggestion
saas-aws-account-operator-bundle/
```

**Comment by 2uasimojo** [comment]
/retest

infra oomkill

**Pull request merged**